.github/workflows/ci.yml

```diff
@@ -44,3 +45,11 @@ jobs:
         run: npm cit
         env:
             CI: true
+            
+      - name: Initialize CodeQL
```

Comment:
It would be better to have this in the `eslint` block above.

The `test` block runs 6 times to test different combinations of 3 operating systems and 2 NodeJS versions. The `eslint` block runs once. We only need to run CodeQL analysis once.